pkgs/applications/science/math/gurobi/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, autoPatchelfHook, python }:
+
+stdenv.mkDerivation rec {
+  name = "gurobi-${version}";
+  version = "8.0.1";
+
+  src = with stdenv.lib; fetchurl {
+    url = "http://packages.gurobi.com/${versions.majorMinor version}/gurobi${version}_linux64.tar.gz";
+    sha256 = "0y3lb0mngnyn7ql4s2n8qxnr1d2xcjdpdhpdjdxc4sc8f2w2ih18";
+  };
+
+  sourceRoot = "gurobi${builtins.replaceStrings ["."] [""] version}/linux64";
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+  buildInputs = [ (python.withPackages (ps: [ ps.gurobipy ])) ];
```

Comment:
This should work by just having `python` and `pythonPackages.gurobipy` here

Reply:
Without `python.withPackages` the Gurobi interactive shell (which is a python script) is not able to find the `gurobipy` module.

Reply:
@jfrankenau Even when `python` itself is in the buildInputs? Interesting, OK.

Reply:
Not that I know of, no.